### DISCUSSION TITLE
Termux: You are likely using a very old version.

### DISCUSSION THREAD

**Discussion by neil-edelman** [termux/termux-packages#13823] [Q&A] [answered]
I am using Termux 0.101, "You are likely using a very old version of Termux, probably installed from the Google Play Store. There are plans to remove . . . disable . . . you are encouraged to move to F-Droid or GitHub . . . ". But I updated '11 months ago' from F-Droid. Also, Google Play seems to have another version. Is there another copy on my phone? How do I remove the Google and just have the F-Droid?

**Comment by agnostic-apollo** [comment, marked as answer]
>I am using Termux 0.101

That is an old version. F-Droid is at `0.118.0`.

https://github.com/termux/termux-app#installation

https://github.com/termux/termux-app#google-play-store-deprecated

**Reply by neil-edelman** [reply]
I figured so. But my F-Droid seems to think it's up-to-date. Should I delete the Google Play version? I'm hesitant to do that because I've made changes such as turning off the auto-complete that I have no idea how to do anymore. Both Google and F-Droid claim ownership of Termux, but I only have one copy, as far as I could tell.

**Reply by sylirre** [reply]
F-Droid and Google Play versions are signed by different keys and can't be installed over each other. Yes, you need to remove Play Store build before installing v0.118 from F-Droid.

**Reply by neil-edelman** [reply]
I uninstalled the Google Play version. Initially, my F-Droid would not install on top of an existing installation, but after about half an hour, it seems to have refreshed and allowed me to install. It still said it was downloading 0.101, but the version I ended up with is 0.118. Thank you!

**Comment by macerg** [comment]
Backup your old system environment before you remove the Google play version. You can restore when the F-Droid termux has been installed.
There are instructions around regarding backup and restore. I'll try and find them for you.

**Comment by macerg** [comment]
Link:-
[Backing up and restoring termux ](https://wiki.termux.com/wiki/Backing_up_Termux)

**Reply by neil-edelman** [reply]
I did _both_ the `termux_backup` script and the tar; I think both were needed.
